playbooks/preflight-check.yml

```diff
@@ -138,3 +138,18 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check space for rootfs backup on infras
+  hosts: "os-infra-hosts"
```

Comment:
Matching the wrong group of hosts, needs an underscore instead of a dash: os-infra_hosts